legal-api/tests/unit/models/test_filing.py

```diff
@@ -672,3 +679,20 @@ def test_linked_not_correction(session):
     # test
     assert filing1.json['filing']['header']['isCorrected'] is False
     assert filing2.json['filing']['header']['affectedFilings'] is not None
+
+
+def test_alteration_filing_with_court_order(session):
```

Comment:
Can you add a test to ensure if any validators are necessary they work?

Reply:
I don't know why but my Flake8 local is slightly different from the GH's action. I moved because Flake8 complianed. How can I make it use the same rules?

Reply:
I added another test that checks what happened when trying to insert invalid data to the columns.